third_party/terraform/resources/resource_compute_instance_group.go

```diff
@@ -108,7 +108,7 @@ func getInstanceReferences(instanceUrls []string) (refs []*compute.InstanceRefer
 
 func validInstanceURLs(instanceUrls []string) bool {
 	for _, v := range instanceUrls {
-		if !strings.HasPrefix(v, "https://www.googleapis.com/compute/v1/") {
+		if !strings.HasPrefix(v, "https://") {
```

Comment:
What do we expect in `instanceUrls` that is NOT a valid instance URL?

Reply:
It's a user-provided value, so we'd expect names or partial URIs instead of self links, which are expected by the API. It's way more likely that users will do that than mistype compute or something. If we checked the base path value, we'd error on (perfectly usable!) URLs for other versions here.